File: AMS.Web/src/app/models/manage-scheme/scheme.ts
export class Scheme 
{
    Id: number;
    Code : string = null;
    Name : string = null;
    Description? : string = null;
    
    CreatedDate? : Date = null;
    CreatedBy? : number = null;
    ModifiedDate? : Date = null;
    ModifiedBy? : number = null;
    IsActive : boolean = true;
}

File: AMS.API/DTO/Scheme/Scheme.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.Scheme
{
    public class Scheme
    {
        public int Id { get; set; }

        public string Code { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public DateTime CreatedDate { get; set; }

        public Int32 CreatedBy { get; set; }

        public DateTime? ModifiedDate { get; set; }

        public Int32 ModifiedBy { get; set; }

        public bool IsActive { get; set; }
    }
}